mypy/messages.py

```diff
@@ -1235,7 +1235,7 @@ def note_call(self,
                   context, code=code)
 
     def unreachable_statement(self, context: Context) -> None:
-        self.fail("Statement is unreachable", context)
+        self.fail("Statement is unreachable", context, code=codes.UNREACHABLE)
 
     def redundant_left_operand(self, op_name: str, context: Context) -> None:
```

Comment:
Can you also include the new error code for the `redundant_blah` error messages listed below here?

These error messages use the same reachability inference logic and are also reported only when the `--warn-unreachable` flag is set.